src/command.coffee

```diff
@@ -185,7 +186,7 @@ compileScript = (file, input, base = null) ->
     else
       compiled = CoffeeScript.compile t.input, t.options
       t.output = compiled
-      if o.map
+      if o.map and not o['inline-map']
```

Comment:
I'm not sure it's possible to have both an inline-map and an external map the browser would just go with whichever was last.